webapp/components/user_settings/user_settings_display.jsx

```diff
@@ -39,6 +39,7 @@ export default class UserSettingsDisplay extends React.Component {
         this.updateSection = this.updateSection.bind(this);
         this.updateState = this.updateState.bind(this);
         this.createCollapseSection = this.createCollapseSection.bind(this);
+        this.isSaving = false;
```

Comment:
This doesn't set state. It sets the property on the object.

Reply:
Actually it's a mistake, I am going to set it as a state property.